cargo-near/src/commands/build_command/docker.rs

```diff
@@ -189,4 +162,82 @@ impl super::BuildCommand {
         };
         Ok((status, docker_cmd))
     }
+
+    const BUILD_COMMAND_CLI_CONFIG_ERR: &'static str =  "cannot be used, when `container_build_command` is configured from `[package.metadata.near.reproducible_build]` in Cargo.toml";
+
+    fn compute_build_command(
+        &self,
+        manifest_command: Option<String>,
+    ) -> color_eyre::eyre::Result<String> {
+        if let Some(cargo_cmd) = manifest_command {
+            if self.no_abi {
+                return Err(color_eyre::eyre::eyre!(format!(
+                    "`{}` {}",
+                    "--no-abi",
+                    Self::BUILD_COMMAND_CLI_CONFIG_ERR
+                )));
+            }
+            if self.no_embed_abi {
+                return Err(color_eyre::eyre::eyre!(format!(
+                    "`{}` {}",
+                    "--no-embed-abi",
+                    Self::BUILD_COMMAND_CLI_CONFIG_ERR
+                )));
+            }
+            if self.no_doc {
+                return Err(color_eyre::eyre::eyre!(format!(
+                    "`{}` {}",
+                    "--no-doc",
+                    Self::BUILD_COMMAND_CLI_CONFIG_ERR
+                )));
+            }
+            if self.no_locked {
+                return Err(color_eyre::eyre::eyre!(format!(
+                    "`{}` {}",
+                    "--no-locked",
+                    Self::BUILD_COMMAND_CLI_CONFIG_ERR
+                )));
+            }
+            if self.no_release {
+                return Err(color_eyre::eyre::eyre!(format!(
+                    "`{}` {}",
+                    "--no-release",
+                    Self::BUILD_COMMAND_CLI_CONFIG_ERR
+                )));
+            }
+            return Ok(cargo_cmd);
+        }
+        println!(
+            " {}",
+            "configuring `container_build_command` from cli args, passed to current command".cyan()
+        );
+        let mut cargo_args = vec![];
+
+        if self.no_abi {
+            cargo_args.push("--no-abi");
+        }
+        if self.no_embed_abi {
+            cargo_args.push("--no-embed-abi");
+        }
+        if self.no_doc {
+            cargo_args.push("--no-doc");
+        }
+        if self.no_locked {
+            return Err(color_eyre::eyre::eyre!("`--no-locked` flag is forbidden for reproducible builds in containers, because a specific Cargo.lock is required"));
+        }
+        if self.no_release {
+            cargo_args.push("--no-release");
+        }
+
+        let color = self
+            .color
+            .clone()
+            .unwrap_or(crate::common::ColorPreference::Auto)
+            .to_string();
+        cargo_args.extend(&["--color", &color]);
```

Comment:
I don't see much value in passing `--color auto` explicitly when it was not specified. Can we conditionally add it there?

```suggestion
        if let Some(color) = self.color {
            cargo_args.extend(&["--color", &color.to_string()]);
        }
```